src/components/Label/Label.stories.js

```diff
@@ -25,3 +26,5 @@ stories.addWithInfo('Label', () => (
     <Label bsStyle="danger">Danger</Label>
   </div>
 ));
+
+stories.addWithInfo('Label with remove', () => <MockLabelRemove />);
```

Comment:
This give very little information in the story book. When using a Mock component, it's likely you will need to specify the `text` and the `propTables` to include example source and the Prop Types for the components you want to document.